x-pack/plugins/cloud_integrations/cloud_data_migration/public/application/components/app.tsx

```diff
@@ -0,0 +1,165 @@
+              />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiTitle size="s">
+                <h3>
+                  <FormattedMessage
+                    id="cloudMigration.migrateToCloudTitle"
+                    defaultMessage="Save time & money by moving your deployment to Elastic Cloud."
+                  />
+                </h3>
+              </EuiTitle>
+
+              <EuiSpacer size="xl" />
+
+              <EuiListGroup maxWidth={700}>
+                <EuiListGroupItem
+                  {...listItemProps}
+                  label={
+                    <FormattedMessage
+                      id="cloudMigration.deployInSeconds.text"
+                      defaultMessage="Deploy in seconds and scale with a click."
+                    />
+                  }
+                />
+
+                <EuiSpacer size="s" />
+
+                <EuiListGroupItem
+                  {...listItemProps}
+                  label={
+                    <FormattedMessage
+                      id="cloudMigration.freeUpEngineering.text"
+                      defaultMessage="Free up your engineering teams from managing the stack."
+                    />
+                  }
+                />
+
+                <EuiSpacer size="s" />
+
+                <EuiListGroupItem
+                  {...listItemProps}
+                  label={
+                    <FormattedMessage
+                      id="cloudMigration.getHelpFromCreators.text"
+                      defaultMessage="Get help from the creators of the stack to help you set up your workload, tune for performance, or scale up to petabytes of data."
+                    />
+                  }
+                />
+
+                <EuiSpacer size="s" />
+
+                <EuiListGroupItem
+                  {...listItemProps}
+                  label={
+                    <FormattedMessage
+                      id="cloudMigration.getInstantAccess.text"
+                      defaultMessage="Get instant access to the latest version of the Elastic Stack with premium features like anomaly detection, searchable snapshots, advanced prevent and protection capabilities, and more."
+                    />
+                  }
+                />
+              </EuiListGroup>
+
+              <EuiSpacer size="l" />
+
+              <div>
+                {/* eslint-disable-next-line @elastic/eui/href-or-on-click */}
+                <EuiButton
+                  fill={true}
+                  fullWidth={false}
+                  href={`${basePath}#/cloud_migration`}
+                  onClick={() => {
+                    console.log('Navigate to info page');
+                  }}
+                >
+                  <FormattedMessage
+                    id="cloudMigration.readInstructionsButtonLabel"
+                    defaultMessage="Read migration instructions"
```

Comment:
```suggestion
                    defaultMessage="Help me move"
```

@gchaps this is another one I'd like your take on. 

Some other options:
- Help me migrate
- Find out how
- Learn more
- I'm ready to move

Reply:
@kellyemurphy I like "Help me move" but does it really help users or just take them to docs.  Also, is it specific enough?

Reply:
Pretty sure it just takes them to the docs so that they can self-service

Reply:
Ok, let's go with "Help me move"